### ISSUE TITLE
cssnano upgraded

### ISSUE THREAD

**Issue by frederikhors** [posthtml/htmlnano#282]
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: htmlnano@2.1.0
npm ERR! Found: cssnano@7.0.1
npm ERR! node_modules/cssnano
npm ERR!   dev cssnano@"7.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional cssnano@"^6.0.0" from htmlnano@2.1.0
npm ERR! node_modules/htmlnano
npm ERR!   dev htmlnano@"2.1.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: cssnano@6.1.2
npm ERR! node_modules/cssnano
npm ERR!   peerOptional cssnano@"^6.0.0" from htmlnano@2.1.0
npm ERR!   node_modules/htmlnano
npm ERR!     dev htmlnano@"2.1.0" from the root project
```

**Comment by maltsev** [comment]
I have released a new version of htmlnano (2.1.1) with cssnano@7. 

Can you try to upgrade to it to see if this error disappears?

**Comment by frederikhors** [comment]
Thanks.
